tests/unit/amo/components/TestStaticPages.js

```diff
@@ -0,0 +1,35 @@
+import { shallow } from 'enzyme';
```

Comment:
We usually keep tests sectioned off for each component; it's easier to find them based on fuzzy search file names that way. I know it's more overhead but I think putting the About and Review Guide tests in their own files will help us keep things organized.